cypress/e2e/po/components/list-row.po.ts

```diff
@@ -14,6 +14,14 @@ export default class ListRowPo extends ComponentPo {
     return this.self().find('.btn.actions');
   }
 
+  accountStatusDisabled() {
+    return this.self().find('.icon-color-red', { timeout: 90000 });
+  }
+
+  accountStatusEnabled() {
+    return this.self().find('.icon-color-green', { timeout: 90000 });
+  }
```

Comment:
these should live in since its specific to the users table `cypress/e2e/po/pages/users-and-auth/users.po.ts`